src/app/shared/directives/highlight.directive.ts

```diff
@@ -0,0 +1,23 @@
+import { Directive, ElementRef, inject, Input, OnChanges } from '@angular/core';
+import { MatOption } from '@angular/material/core';
+
+@Directive({
+  selector: 'mat-option[appHighlight]',
+  standalone: true,
+})
+export class HighlightDirective implements OnChanges {
```

Comment:
możemy w sumie wskazać w nazwie że dotyczy to MatOption

Reply:
poprawione